packages/python-packages/protocol-stub-generator/setup.py

```diff
@@ -0,0 +1,48 @@
+from setuptools import setup, find_packages
+import os, re
+
+PACKAGE_NAME = "protocol-stub-generator"
+
+DESCRIPTION = (
+    "A llc stub generator for published APIs, variables and properties in a package"
+)
+
+with open(os.path.join("protocol", "_version.py"), "r") as fd:
```

Comment:
This is looking for `_version.py` but the file is called `version.py`. I would rename the file with the underscore prefix.